schemas/json/theme.json

```diff
@@ -2331,12 +2331,6 @@
 										{
 											"type": "object",
 											"properties": {
-												"source": {
```

Comment:
Source isn't required right now in theme.json as we assume values are paths to files (relative or absolute)